project/CMakeLists.txt

```diff
@@ -148,10 +147,9 @@ set(ALL_TARGETS
     berkeleydb
     bison
     boost
-    breakpad
+    #breakpad
```

Comment:
Does this library not support shared libraries? After removing it, did nebula-ng stop using it in the future?

Reply:
This library was introduced to reduce the transfer time of the user coredump, and if we decide not to use it again in the future, some of the package scripts and code in nebula-ng will also need to be modified